.scala-steward.conf

```diff
@@ -8,5 +8,7 @@ updates.pin = [
   # simpleclient-0.12 breaks metric names
   { groupId = "io.prometheus", version = "0.11." },
   # okio-3 has (slight) binary incompatibilities
-  { groupId = "com.squareup.okio", artifactId = "okio", version = "2." }
+  { groupId = "com.squareup.okio", artifactId = "okio", version = "2." },
+  # scalatags-0.11 breaks bincompat
+  { groupId = "com.lihaoyi", artifactId = "scalatags", version = "0.10." },
```

Comment:
We'll want to remember to remove this one on main.

Reply:
I've added #5654 to track this.